scripts/core/editor3/reducers/editor3.jsx

```diff
@@ -55,20 +55,32 @@ export const forceUpdate = (state) => {
 /**
  * @ngdoc method
  * @name onChange
+ * @param {Object} state
  * @param {Object} editorState
+ * @param {Bool} update if true force change value and editor refresh
```

Comment:
Let's call this parameter `force`. It makes it obvious that it _forces_ the onChange operation. Maybe we can document it as such:

"_When true, forces an editor update regardless of whether the content has changed. This is used because currently it is impossible to detect changes happening solely on entity data. In Draft v0.11.0 we will be able to request all entities from the content and could compare them to get a more accurate result.
See https://draftjs.org/docs/api-reference-content-state.html#getentitymap_"

Then we can remove the comments further down in the code because this explains everything.